sdk/nodejs/connect.ts

```diff
@@ -6,8 +6,25 @@ import { Writable } from "node:stream"
  * ConnectOpts defines option used to connect to an engine.
  */
 export interface ConnectOpts {
+  /**
+   * Use to override Dagger workdir
+   */
   Workdir?: string
+  /**
+   * Use to override Dagger config
+   */
   ConfigPath?: string
+  /**
+   * Enable logs output
+   * @example
+   * LogOutput
+   * ```ts
+   * connect(async (client: Client) => {
+    const source = await client.host().workdir().id()
+    ...
+    }, {LogOutput: process.stdout})
+    ```
+   */
```

Comment:
We should add a test too on this

Reply:
> We should add a test too on this

Good point.
This should be done in a follow up PR. Would you like to take it ?

Reply:
I'm down to take it! Let's rebase & merge this PR so I can work on it :D